frontend/src/components/Input/index.module.css

```diff
@@ -0,0 +1,44 @@
+.errorMessage {
+  padding: 0;
+  margin: 0;
+  margin-bottom: var(--font-size-xxs);
```

Comment:
The variable is called `font-size` but we're using it for `margin` too?

Reply:
Check my other reply, this was me, we'll do a follow up renaming the variables